src/arrays/arraymath.jl

```diff
@@ -90,6 +90,12 @@ Base.scale(qarr::Union(QuArray,CTranspose), num::Number) = scale(num, qarr)
 *(qarr::Union(QuArray,CTranspose), num::Number) = scale(qarr, num)
 /(qarr::Union(QuArray,CTranspose), num::Number) = scale(1/num, qarr)
 
+# sparse to dense
+Base.full(qarr::AbstractQuMatrix) = QuArray(full(coeffs(qarr)),bases(qarr))
+
+# exponential of dense matrix
+Base.expm(qarr::AbstractQuMatrix) = QuArray(expm(coeffs(full(qarr))),bases(qarr))
```

Comment:
Wouldn't it be easier to just say `expm(full(coeffs(qarr)))`? But maybe it doesn't matter.